### ISSUE TITLE
Cursor jumps to first column

### ISSUE THREAD

**Issue by laktak** [sunaku/vim-shortcut#3]
Whenever I run `:Shortcuts` and then select a command or just press `ESC` the cursor jumps to the first column.

My commands mostly match your samples:

```
  Shortcut Python-goto map <Leader>ipg :call jedi#goto()<CR>
  Shortcut Python-documentation map <Leader>ipd :call jedi#show_documentation()<CR>
```

Any idea what I'm doing wrong?

**Comment by sunaku** [comment]
Good observation! :+1:

The root cause of this issue is the `<line1>,<line2>` range :scream: that is passed along to the **call** command:

```vim
command! -range -bang Shortcuts <line1>,<line2>call s:shortcut_menu_command(<bang>0)
```

Removing that range prevents this issue from happening, but it also throws away useful functionality. :sweat:

**Comment by laktak** [comment]
Well the problem is that some of my commands need the word under the cursor (like for 'lookup definition' etc) and can't get it because of this.

Do you think you can find a workaround or introduce a second Shortcuts command without the range?

**Comment by sunaku** [comment]
For now, please consider applying that workaround locally by editing your copy of this plugin's code. :hammer_and_wrench:

To really fix this issue, [we need FZF.vim to provide an "at exit" hook](https://github.com/junegunn/fzf.vim/issues/369) and no such hook currently exists. :disappointed:

**Comment by laktak** [comment]
Thanks for looking into this but wouldn't a shortcut command without range for `nmap` and a shortcut command with range for `vmap` solve this as well?

**Comment by sunaku** [comment]
A pair of secondary commands would indeed solve the issue as described. :cold_sweat: However, they wouldn't be a solution as the target of keybindings: I _sometimes_ specify a count before issuing key bindings (e.g. 15 followed by "increase window size" performs the equivalent of issuing that shortcut 15 times repeatedly) and the range parameter supports both that occasional usage and also the more frequent countless usage.

**Comment by sunaku** [comment]
@laktak Please try the `issue-3` branch which adds `:ShortcutsRangeless` commands to fix this issue.

**Comment by sunaku** [comment]
Fix merged into master at commit ee18931b6fffc14f04018af1176c55ae1b539d37.

**Comment by laktak** [comment]
@sunaku thanks but I'm not using the extension at the moment.
